test/src/main/java/org/apache/accumulo/test/compaction/CompactionPriorityQueueMetricsIT.java

```diff
@@ -202,7 +202,7 @@ public void configureMiniCluster(MiniAccumuloConfigImpl cfg, Configuration conf)
           Property.COMPACTION_SERVICE_PREFIX.getKey() + QUEUE1_SERVICE + ".planner.opts.groups",
           "[{'group':'" + QUEUE1 + "'}]");
 
-      cfg.setProperty(Property.MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_INITIAL_SIZE, "6");
+      cfg.setProperty(Property.MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_SIZE, "6");
```

Comment:
I think the previous value was 6 entries, now it's 6 bytes. Is this too small?

Reply:
Yeah that is too small will change it to something more reasonable and change the test if needed.